.prerequisites.py

```diff
@@ -45,37 +42,36 @@ def vault_version_checker(_cmd):
 
     return str(installed_version)
 
+# Checks that need to be run on all operating systems.
 both_checks = [
     ('git',
-     {'osx': 'brew install git'}),
+     {'Linux': 'apt-get install build-essential',
+      'Mac OS': 'xcode-select --install'}),
 
     ('virtualenv',
-     {'all': 'sudo pip install virtualenv'}),
+     {'Linux': 'sudo pip install virtualenv',
+      'Mac OS': 'brew install python@2'}),
 
     # needed for installing pynacl, which is a transitive dependency of Paramiko which is a dependency of Fabric
     ('make',
-     {'all': 'which make'},
+     {'Linux': 'apt-get install build-essential',
+      'Mac OS': 'xcode-select --install'},
      dumb_install_check,
      lambda x: shs('make -v').splitlines()[0]),
 
     ('virtualbox',
-     {'osx': 'brew cask install virtualbox'},
+     {'osx': 'brew install --cask virtualbox'},
      dumb_install_check,
      lambda x: shs('vboxmanage --version')),
 
     ('vagrant',
-     {'osx': 'brew cask install vagrant'}),
+     {'osx': 'brew install --cask vagrant'}),
```

Comment:
do you want `Mac OS` here rather than `osx`?

Reply:
Good spot, fixed